client-app/src/examples/portfolio/detail/DetailPanelModel.js

```diff
@@ -1,17 +1,16 @@
-import {HoistModel, loadAllAsync, managed} from '@xh/hoist/core';
-import {bindable, makeObservable} from '@xh/hoist/mobx';
+import {HoistModel, lookup, managed} from '@xh/hoist/core';
+import {observable, makeObservable} from '@xh/hoist/mobx';
 import {PanelModel} from '@xh/hoist/desktop/cmp/panel';
 import {OrdersPanelModel} from './OrdersPanelModel';
-import {ChartsPanelModel} from './ChartsPanelModel';
 import {PERSIST_DETAIL} from '../AppModel';
-
+import {PortfolioPanelModel} from '../PortfolioPanelModel';
 
 export class DetailPanelModel extends HoistModel {
 
-    @bindable positionId = null;
+    @observable positionId = null;
 
-    @managed ordersPanelModel = new OrdersPanelModel();
-    @managed chartsPanelModel = new ChartsPanelModel();
+    @lookup(PortfolioPanelModel) parentModel;
+    @managed ordersPanelModel = new OrdersPanelModel(this);
```

Comment:
This looks like the anti-pattern to the change here. Aren't we just create this here so that it can have a reference to the parent? Shouldn't OrdersPanelModel just lookUp the DetailPanelModel?

Reply:
Maybe anti-pattern isn't right, but rather "the old way"

Reply:
this was an example where parent actually needed a reference to the OrdersPanelModel.  could have done it in a number of ways, chose to leave this one like this.